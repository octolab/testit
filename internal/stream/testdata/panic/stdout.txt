FAIL	testdata/nocompile [build failed]
?   	testdata/nofiles	[no test files]
ok  	testdata/notests	0.643s
--- FAIL: TestTheGood_Divide (0.00s)
--- FAIL: TestTheBad_Divide (0.00s)
panic: division by zero [recovered]
	panic: division by zero

goroutine 19 [running]:
testing.tRunner.func1.1(0x111ba60, 0x116cff0)
	$go/libexec/src/testing/testing.go:1072 +0x30d
testing.tRunner.func1(0xc000082900)
	$go/libexec/src/testing/testing.go:1075 +0x41a
panic(0x111ba60, 0x116cff0)
	$go/libexec/src/runtime/panic.go:969 +0x1b9
testdata/panicked.TheBad.Divide(...)
	$testit/internal/testdata/panicked/panicked.go:19
testdata/panicked.TestTheBad_Divide(0xc000082900)
	$testit/internal/testdata/panicked/panicked_test.go:13 +0x39
testing.tRunner(0xc000082900, 0x114df50)
	$go/libexec/src/testing/testing.go:1123 +0xef
created by testing.(*T).Run
	$go/libexec/src/testing/testing.go:1168 +0x2b3
FAIL	testdata/panicked	0.443s
FAIL
