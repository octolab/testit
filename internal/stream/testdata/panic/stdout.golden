FAIL	testdata/nocompile [build failed]
?   	testdata/nofiles	[no test files]
ok  	testdata/notests	0.643s
--- FAIL: TestTheGood_Divide (0.00s)
--- FAIL: TestTheBad_Divide (0.00s)
panic: division by zero [recovered]
	panic: division by zero

1: running[0;90m [Created by testing.(*T).Run @ /usr/local/Cellar/go/1.15.6/libexec/src/testing/testing.go:1168][39m[m
    [0;1;39mtesting  [39m[m/usr/local/Cellar/go/1.15.6/libexec/src/testing/testing.go:1072                             [0;1;32mtRunner.func1.1[39m[m(*T(#1), func(#2))[39m[m
    [0;1;39mtesting  [39m[m/usr/local/Cellar/go/1.15.6/libexec/src/testing/testing.go:1075                             [0;32mtRunner.func1[39m[m(*T(#3))[39m[m
    [0;1;39m         [39m[m/usr/local/Cellar/go/1.15.6/libexec/src/runtime/panic.go:969                                [0;32mpanic[39m[m(interface{}(#1))[39m[m
    [0;1;39mpanicked [39m[m/Users/ksamigullin/Development/public/testit/internal/testdata/panicked/panicked.go:19      [0;1;31mTheBad.Divide[39m[m(...)[39m[m
    [0;1;39mpanicked [39m[m/Users/ksamigullin/Development/public/testit/internal/testdata/panicked/panicked_test.go:13 [0;1;31mTestTheBad_Divide[39m[m(*T(#3))[39m[m
    [0;1;39mtesting  [39m[m/usr/local/Cellar/go/1.15.6/libexec/src/testing/testing.go:1123                             [0;32mtRunner[39m[m(*T(#3), func(0x114df50))[39m[m
FAIL	testdata/panicked	0.443s
FAIL
